app/models/refinery/blog/category.rb

```diff
@@ -4,6 +4,9 @@ class Category < ActiveRecord::Base
       extend FriendlyId
 
       translates :title, :slug
+      attribute :title
```

Comment:
Nice! Attributes API. I think it would be valuable to also specify the types for these ( http://jakeyesbeck.com/2015/12/20/rails-5-attributes/ ) what do you think?

Reply:
Done!  Still stalking some of the other errors.